airflow/www/static/js/dag/details/Dag.tsx

```diff
@@ -291,6 +292,19 @@ const Dag = () => {
                   )}
                 </Td>
               </Tr>
+              <Tr>
+                <Td>Dag run timeout</Td>
+                <Td>
+                  {dagDetailsData.dagRunTimeout?.type === undefined ? (
+                    <Text>null</Text>
+                  ) : (
+                    // for TimeDelta and RelativeDelta
+                    <ViewScheduleInterval
```

Comment:
`ViewScheduleInterval` couples this component with dag's schedule interval. Maybe this could be refactored/renamed to `ViewTimeDelta` component to be used in schedule interval and dag run timeout to render timedelta values?

Reply:
Makes sense. Let me take a look and make the change. Cheers.